Mention dependence on async-trait

Without `async-trait` as a dependency of my project it was failing to compile with the below message. I saw the example depended on it even though it was never explicitly included so I decided to try adding it, and it fixed the compiler error.

Possibly hugsqlx should provide it, or make it explicit the user must bring it into scope in files that use the `HugSqlx` trait :)

```
error[E0706]: functions in traits cannot be declared `async`
 --> src/fun.rs:3:10
  |
3 | #[derive(HugSqlx)]
  |          ^^^^^^^ `async` because of this
  |
  = note: `async` trait functions are not currently supported
  = note: consider using the `async-trait` crate: https://crates.io/crates/async-trait
  = note: this error originates in the derive macro `HugSqlx` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: use of undeclared crate or module `async_trait`
 --> src/fun.rs:3:10
  |
3 | #[derive(HugSqlx)]
  |          ^^^^^^^ use of undeclared crate or module `async_trait`
  |
  = note: this error originates in the derive macro `HugSqlx` (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/README.md
+++ b/README.md
@@ -4,10 +4,16 @@ HugSQLx is a derive macro turning SQL queries into plain Rust functions. This is
 
 ## Installation
 
+HugSQLx stands on a shoulders of 2 other crates: [async_trait](https://crates.io/crates/async-trait) and [SQLx](https://crates.io/crates/sqlx):
+
 ``` toml
 [dependencies]
-hugsqlx="0.1.2" 
+async-trait = "0.1.58"
+sqlx = { version = "0.6.2", features = ["sqlite"] }
+hugsqlx = { version = "0.1.2", features = ["sqlite"] }
 ```
+
+Both HugSQLx and SQLx itself should have the same database mentioned in *features* (sqlite, postgres or mysql).
 
 ## Deep dive into named queries
 The idea here is to distinguish 3 types of queries:

